frontend/src/components/DecayInformation.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <span v-if="decaytyp === 'short'">
-      <small>{{ decay ? ' ' + decay.balance + ' ' + decayStartBlockTextShort : '' }}</small>
+      <small>{{ decay ? ' -' + decay.balance + ' ' + decayStartBlockTextShort : '' }}</small>
```

Comment:
![image](https://user-images.githubusercontent.com/15882241/125961121-65ada3f9-5f2c-475f-8a12-65b153dab034.png)

Das `to` ist Wiederhlulng. Wir bauchen weder `to` noch `from`

Reply:
das ist im #630 auch erledigt